components/search/SearchQueryOptions.vue

```diff
@@ -0,0 +1,426 @@
+<template>
+  <b-list-group
+    :id="elementId"
+    class="auto-suggest-dropdown"
+    data-qa="search suggestions"
+    role="listbox"
+    :aria-label="$t('searchSuggestions')"
+    :aria-hidden="!isActive"
+  >
+    <template
+      v-if="onCollectionPage"
+    >
+      <b-list-group-item
```

Comment:
These list-group-items are not the same, but look similar. I am wondering if the code can be more DRY?

Reply:
each item is already using a component, and there are subtle differences between the various elements. So the I'm not sure how this could be refactored.